packages/components/src/utils.ts

```diff
@@ -1,13 +1,15 @@
 import { createIntl } from 'react-intl'
-import snarkdown from 'snarkdown'
```

Comment:
**note**: snarkdown is quite outdated, so we're replacing it with MarkdownIt which supports converting plain URLs to links.